notebooks/book2/Fig16_1_(book2)_mlpPriorsDemo.ipynb

```diff
@@ -0,0 +1,998 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.        for i, lyr in enumerate(self.layers):
</pre><p>We can avoid using a for loop here because this is a demo and we have only one hidden layer. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>